app/actions/WalletLoaderActions.js

```diff
@@ -45,38 +45,40 @@ export function walletExistRequest() {
   return (dispatch, getState) => {
     dispatch({ type: WALLETEXIST_ATTEMPT });
     const { loader } = getState().walletLoader;
-    loader.walletExists(request,
-        function(error, response) {
-          if (error) {
-            dispatch({ error, type: WALLETEXIST_FAILED });
-          } else {
-            dispatch({response: response, type: WALLETEXIST_SUCCESS });
-            if (response.getExists()) {
-              setTimeout(dispatch(openWalletAttempt("public")), 1000);
-            }
-          }
-        });
+    loader.walletExists(request, function(error, response) {
+      if (error) {
+        dispatch({ error, type: WALLETEXIST_FAILED });
+      } else {
+        dispatch({response: response, type: WALLETEXIST_SUCCESS });
+        if (response.getExists()) {
+          setTimeout(() => dispatch(openWalletAttempt("public", false)), 1000);
```

Comment:
@alexlyp I fixed a bug here; this wasn't wrapped in a function so it's actually calling `dispatch()` immediately. Can the `setTimeout()` call be removed?